src/utils/mesh_utils.ts

```diff
@@ -16,7 +16,7 @@ import { orderUtils } from './order_utils';
 
 export const meshUtils = {
     orderInfosToApiOrders: (
-        orderEvent: Array<OrderEvent | AcceptedOrderInfo | RejectedOrderInfo | OrderInfo>,
+        orderEvent: (OrderEvent | AcceptedOrderInfo | RejectedOrderInfo | OrderInfo)[],
```

Comment:
I feel like VSCode always yelledd at me for this syntax over `Array<X>` 😂

Reply:
Yeah they changed it to prefer `Something[]` in v4 🤷